src/config/jest-setup.js

```diff
@@ -1,63 +0,0 @@
-/* eslint-env jest */
```

Comment:
Can we verify that none of the existing repos in our orgs are using jest somehow?

Reply:
It's been verified